pallets/vesting/src/weights.rs

```diff
@@ -2,12 +2,13 @@
 //! Autogenerated weights for `pallet_vesting`
 //!
 //! THIS FILE WAS AUTO-GENERATED USING THE SUBSTRATE BENCHMARK CLI VERSION 4.0.0-dev
-//! DATE: 2022-03-01, STEPS: `20`, REPEAT: 64, LOW RANGE: `[]`, HIGH RANGE: `[]`
+//! DATE: 2022-05-25, STEPS: `20`, REPEAT: 64, LOW RANGE: `[]`, HIGH RANGE: `[]`
```

Comment:
No idea if I did this right. Just ran the command in comments and hoped for the best. The number of reads/writes look reasonable but I'm not sure how to make sense of the const values.